tests/scripts/check-files.py

```diff
@@ -181,10 +181,23 @@ def __init__(self, log_file):
         self.logger = None
         self.setup_logger(log_file)
         self.extensions_to_check = (
-            ".c", ".h", ".sh", ".pl", ".py", ".md", ".function", ".data",
-            "/Makefile", "/Makefile.inc", "/CMakeLists.txt", "/ChangeLog"
+            ".c",
+            ".h",
+            ".sh",
+            ".pl",
+            ".py",
+            ".md",
+            ".function",
+            ".data",
+            "/Makefile",
+            "/Makefile.inc",
+            "/CMakeLists.txt",
+            "/ChangeLog",
```

Comment:
`ChangeLog `shouldn't have a comma after it in *this* commit, since here it is the last element of the tuple. It only needs a comma in the next commit.

Reply:
Yes, it should have a comma in this commit. Having a comma after the last element of a multiline list is recommended, precisely in case the element ends up not being the last one anymore.

Reply:
Ok.